dolly-backend-app/src/main/java/no/nav/dolly/provider/api/OppslagController.java

```diff
@@ -70,6 +71,13 @@ public KodeverkAdjusted fetchKodeverkByName(@PathVariable("kodeverkNavn") String
         return kodeverkMapper.mapBetydningToAdjustedKodeverk(kodeverkNavn, response.getBetydninger());
     }
 
+    @GetMapping("/kodeverk")
+    @Operation(description = "Hent kodeverk, returnerer map")
+    public Map<String, String> fetchKommuner(@RequestParam String kodeverk) {
```

Comment:
Kanskje endre fetchKommuner til et mer generelt navn, for dette vil vel fungere for flere kodeverk ikkesant? 😄

Reply:
will do